packages/ui/src/views/repo/pull-request/components/reviewers/pull-request-reviewers-header.tsx

```diff
@@ -28,78 +30,79 @@ const ReviewersHeader = ({
   useTranslationStore
 }: ReviewersHeaderProps) => {
   const { t } = useTranslationStore()
-  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchQuery(event.target.value)
-  }
+
+  const { search, handleSearchChange, handleResetSearch } = useDebounceSearch({
+    handleChangeSearchValue: setSearchQuery,
+    searchValue: searchQuery
+  })
+
+  const handleCloseValuesView = useRef(debounce(() => handleResetSearch(), 300)).current
```

Comment:
Should be safe to remove the inline anonymous function

```suggestion
  const handleCloseValuesView = useRef(debounce(handleResetSearch, 300)).current
```